src/cf.data.pre

```diff
@@ -4868,6 +4868,28 @@ DOC_START
 			values may significantly understate or exaggerate actual times.
 			Do not use this measurement unless you know it works in your case.
 
+TODO: Reduce duplication??
+XXX: Format.
+
+event_time{name}: Approximate time of the named event in <full seconds since epoch>.<fractional seconds> format. The following events are currently supported:
+
```

Comment:
For now, all %events show the time in absolute milliseconds, but we can adjust to the 'fractional seconds', if needed.

Reply:
Done (bf09e96).